endpoints/helpline-search.php

```diff
@@ -55,9 +55,7 @@
     $extension = isset($exploded_result[1]) ? $exploded_result[1] : "w";
     $service_body_id = isset($service_body_obj) ? $service_body_obj->id : 0;
 
-if (!setting("tomato_helpline_routing") && !isset($_REQUEST['ForceNumber'])) {
```

Comment:
Why are you removing this?

Reply:
With that on it was catching the tomato_helpline_routing and overriding if it was discovered that the rootserver was owned

Reply:
re-write so it still does not execute this line:

```
if (!isset($_REQUEST['ForceNumber'])) {
    $serviceBodyCallHandling = getServiceBodyCallHandling($service_body_id);
}
```

Reply:
Will do 

Will need to be later - won’t be near my computer until this evening.